Automattic/jetpack
# Implement correct table interactions on the feedback dashboard

## Proposed changes:

This patch fixes some of the issues with the table on the feedback dashboard introduced in #29007 and adds remaining missing interactions.

As things are right now, you should be able to use the checkboxes for selecting multiple responses and performing actions, while clicking on any row by itself will expand that response on the right. The currently expanded response is also highlighted in the table.

### Other information:


## Jetpack product discussion



pcsBup-Pl-p2

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






Make sure you have the jetpack forms package and new dashboard enabled with these filters:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

You'll also need to already have gathered some feedback for it to show up in the dashboard :)

- Go to `/wp-admin/admin.php?page=jetpack-forms`, where you should see the new dashboard.
- On the left you should see the table with the current set of results, and on the right an expanded view of the currently active item.
- Clicking any row on the left, should update the view on the right with the information from that row.
- The currently active row should be highlighted compared to other rows.
- You should be able to select/deselect checkboxes independently of what response is currently 'active'.
